### PR TITLE
Ankit |Test | PROD - E-Invoice column is showing in the get all page even when setting is off for the E-Invoice

### PR DESCRIPTION
**Pull request by ankitdubeyoffice**
https://app.clickup.com/t/86cwxrtrd
https://app.clickup.com/t/86cwx1v40

## Summary by CodeRabbit

- **New Features**
	- Enhanced aging report interface for improved layout and responsiveness.
	- Duplicated "Create New Customer" button for mobile view.
	- Improved search functionality with clearer event handling.
	- Dynamic display of voucher columns based on invoice settings.
	- Updated pagination logic to display only when necessary.

- **Bug Fixes**
	- Refined logic for displaying messages in no-data states.

- **Style**
	- Updated CSS for the aging report and contact management interfaces to improve stacking context and overall responsiveness.
	- Introduced new CSS classes for better visual hierarchy and responsiveness.

### CODE REVIEW

File: apps/web-giddh/src/app/vouchers/list/list.component.ts [hunk 1]
```diff
@@ -613,6 +613,30 @@ export class VoucherListComponent implements OnInit, OnDestroy {
...
+                        this.displayedColumns = this.displayedColumns?.filter(column => column !== "einvoicestatus");
+                    }
+                } else if (this.voucherType === VoucherTypeEnum.purchase) {
+                    this.applyRoundOff = settings.invoiceSettings.purchaseRoundOff;
+                } else if (this.voucherType === VoucherTypeEnum.debitNote) {
+                    this.applyRoundOff = settings.invoiceSettings.debitNoteRoundOff;
+                } else if (this.voucherType === VoucherTypeEnum.creditNote) {
+                    this.applyRoundOff = settings.invoiceSettings.creditNoteRoundOff;
+                } else if (this.voucherType === VoucherTypeEnum.estimate || this.voucherType === VoucherTypeEnum.generateEstimate || this.voucherType === VoucherTypeEnum.proforma || this.voucherType === VoucherTypeEnum.generateProforma) {
+                    this.applyRoundOff = true;
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/vouchers/list/list.component.ts:681]: please check again

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/vouchers/list/list.component.ts (hunk 1 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/vouchers/list/list.component.ts:681]: Done

**Review by dvCodeWorld — APPROVED** [review]

File: apps/web-giddh/src/app/vouchers/list/list.component.ts [hunk 2]
```diff
@@ -613,6 +613,33 @@ export class VoucherListComponent implements OnInit, OnDestroy {
                 this.allVouchersSelected = false;
             }
         });
+
+        this.componentStore.invoiceSettings$.pipe(takeUntil(this.destroyed$)).subscribe(settings => {
+            if (settings) {
+
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/vouchers/list/list.component.ts]: remove white space

File: apps/web-giddh/src/app/vouchers/list/list.component.ts [hunk 3]
```diff
@@ -324,7 +322,6 @@ export class VoucherListComponent implements OnInit, OnDestroy {
                 this.company.giddhBalanceDecimalPlaces = response.balanceDecimalPlaces;
             }
         });
-        this.store.dispatch(this.settingsProfileActions.GetInventoryInfo());
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/vouchers/list/list.component.ts:327]: please check this

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

**Comment by ankitdubeyoffice** [comment]
@coderabbitai.  review

File: apps/web-giddh/src/app/vouchers/list/list.component.ts [hunk 4]
```diff
@@ -347,9 +345,7 @@ export class VoucherListComponent implements OnInit, OnDestroy {
         private invoiceService: InvoiceService,
         private adjustmentUtilityService: AdjustmentUtilityService,
         private salesAction: SalesActions,
-        private settingsProfileActions: SettingsProfileActions,
-        private invoiceActions: InvoiceActions,
-        private commonActions: CommonActions
+        private invoiceActions: InvoiceActions
```
[the remark this answers is not in the document]
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/vouchers/list/list.component.ts:348]: this.invoiceActions.GetAllLedgersForInvoice(
                this.prepareQueryParamsForLedgerApi(),
                this.prepareModelForLedgerApi()
            )

[the remark this answers is not in the document]
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/vouchers/list/list.component.ts:348]: this.invoiceActions.GetAllLedgersForInvoice(
                this.prepareQueryParamsForLedgerApi(),
                this.prepareModelForLedgerApi()
            )

**Comment by ankitdubeyoffice** [comment]
@coderabbitai. review

**Review by dilpreetsinghofficial — APPROVED** [review]

**Pull request merged**